docker_templates/templates/docker_images_legacy/create_ros_core_image.Dockerfile.em

```diff
@@ -29,10 +29,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 @[  end if]@
 @[end if]@
 # setup keys
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 421C365BD9FF1F717815A3895523BAEEB01FA116
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys AD19BAB3CBF125EA
```

Comment:
I see the [ROS wiki](http://wiki.ros.org/action/fullsearch/SnapshotRepository) uses a slightly different invocation. Is there one to prefer to the other? if yes we could homogenize so that here and the wiki use the same.
`apt-key adv --keyserver keyserver.ubuntu.com --recv-key 0xCBF125EA`

Reply:
As long as the apt version installed with the legacy Ubuntu distro supports the input or the entire key as apposed to a truncation, I'd prefer the entire key.

Reply:
Sounds good :+1:

> As long as the apt version installed with the legacy Ubuntu distro supports the input or the entire key

This passed the build on lucid and bionic so it should be good